terragrunt/modules/gha-self-hosted-images/cdn.tf

```diff
@@ -0,0 +1,106 @@
+locals {
+  domain_name = "gha-self-hosted-images.infra.rust-lang.org"
+}
+
+resource "aws_cloudfront_distribution" "cdn" {
+  comment = local.domain_name
+  aliases = [local.domain_name]
+
+  enabled             = true
+  wait_for_deployment = false
+  is_ipv6_enabled     = true
+  price_class         = "PriceClass_All"
+
+  viewer_certificate {
+    acm_certificate_arn      = module.certificate.arn
+    ssl_support_method       = "sni-only"
+    minimum_protocol_version = "TLSv1.1_2016"
```

Comment:
Can you bump this up to something more modern? Ideally the most recently defined policy.